### DISCUSSION TITLE
I can help with translation to danish, if thats of interest?

### DISCUSSION THREAD

**Discussion by Krathlusker** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#422] [Ideas]
Im not much of a techguru, and i dont know much about coding. But i do use this brilliant software on my desktop! I would love to help you guys translate the software to danish 😄

**Comment by sumeshir26** [comment]
Hello @Krathlusker,
Even if we tried, we cannot add that language because it is not supported by the Google Assistant SDK. You can find the supported languages [here](https://developers.google.com/assistant/sdk/reference/rpc/languages).
Thank you for asking!

**Comment by marc873a** [comment]
Hi @Krathlusker and @sumeshir26 
I'm also from DK and I tried the app today. I don't know why but as I spoke to her, she changed to the Danish language. Now she understands and speaks danish. Maybe it's because I use her on my phone too ¯\_(ツ)_/¯

**Reply by sumeshir26** [reply]
@marc873a The Google Assistant _does_ support Danish but the Developer SDK does not unfortunately.

**Reply by marc873a** [reply]
@sumeshir26 I'm not a tech guy or anything so I don't know what the dev SDK is.

All I know is that it works on danish for me (And I'm happy about it)

**Comment by obed9057** [comment]
It means that you cant use developer mode in Danish because javascript cant be transalated properly Sent from Mail for Windows 10 From: marc873aSent: June 16, 2021 3:12 PMTo: Melvin-Abraham/Google-Assistant-Unofficial-Desktop-ClientCc: SubscribedSubject: Re: [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client] I can help with translation to danish, if thats of interest? ***@***.*** I'm not a tech guy or anything so I don't know what the dev SDK is.All I know is that it works on danish for me (And I'm happy about it)—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Reply by marc873a** [reply]
@obed9057 What exactly are you writing here

**Comment by obed9057** [comment]
You said you dont know what the dev sdk is well I just did explain it Sent from Mail for Windows 10 From: marc873aSent: June 16, 2021 3:38 PMTo: Melvin-Abraham/Google-Assistant-Unofficial-Desktop-ClientCc: obed9057; MentionSubject: Re: [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client] I can help with translation to danish, if thats of interest? ***@***.*** What exactly are you writing here—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by Krathlusker** [comment]
I am aware that she speaks and listens to danish - i was offering to
translate the app :-)

Den ons. 16. jun. 2021 kl. 21.49 skrev obed9057 ***@***.***>:

**Reply by Melvin-Abraham** [reply]
@Krathlusker This would definitely come in handy once a proper i18n solution is implemented...
